ssl/t1_lib.c

```diff
@@ -479,25 +479,29 @@ static int add_provider_sigalgs(const OSSL_PARAM params[], void *data)
     if (sinf->algorithm == NULL)
         goto err;
 
-    /* Truly optional */
+    /* Optional */
     p = OSSL_PARAM_locate_const(params, OSSL_CAPABILITY_TLS_SIGALG_HASHALG);
-    if (p == NULL || p->data_type != OSSL_PARAM_UTF8_STRING) {
+    if (p == NULL)
         sinf->hash_algorithm = NULL;
-    }
+    else if (p->data_type != OSSL_PARAM_UTF8_STRING)
```

Comment:
Since one else branch uses `{}` our coding style says that all branches should use `{}` even if they are only one line.

Reply:
So done in all places in https://github.com/openssl/openssl/pull/19312/commits/8be6985a9743a239684308b6258691d3489e52bf

Reply:
This doesn't seem to be actually done?

Reply:
Ah, @mattcaswell is correct.  Since the `else` body a couple of lines down is bracketed, *all* of the `if` ... `else if` ... `else` bodies must be bracketed.

Reply:
now done